flake8_builtins.py

```diff
@@ -34,6 +38,20 @@ def add_options(cls, option_manager):
             comma_separated_list=True,
             help='A comma separated list of builtins to skip checking',
         )
+        option_manager.add_option(
+            '--builtins-allowed-modules',
+            metavar='builtins',
+            parse_from_config=True,
+            comma_separated_list=True,
+            help='A comma separated list of builtin module names to allow',
+        )
+        option_manager.add_option(
+            '--builtins-allowed-modules',
+            metavar='builtins',
+            parse_from_config=True,
+            comma_separated_list=True,
+            help='A comma separated list of builtin module names to allow',
+        )
```

Comment:
I'm tired or the same option is defined twice? 🤔

Reply:
This looks like bad conflict resolution, I had to do a lot of fixups, since my editor was set to format automatically and I had to fix up a load of double quotes -> single quote :cry:

Reply:
Addressed in 33bc80d8c52a65da7193af83812860a04cd8dfd2